iceoryx_hoofs/test/moduletests/test_unit_duration.cpp

```diff
@@ -1647,6 +1647,68 @@ TEST(Duration_test, AddDurationResultsInSaturationFromSeconds)
     EXPECT_THAT(sut2, Eq(DurationAccessor::max()));
 }
 
+TEST(Duration_test, AddAssignSecondsToDurationResultsInSecondsAdditionToLHS)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "c223f8b3-d396-4c45-8c2d-b6b7d7b7f57a");
+
+    constexpr Duration EXPECTED_DURATION = createDuration(3U, 0U);
+    auto sut = createDuration(2U, 0U);
+    auto otherDuration = createDuration(1U, 0U);
+
+    sut += otherDuration;
+
+    EXPECT_THAT(sut, Eq(EXPECTED_DURATION));
```

Comment:
@ithier No need to change, but I prefer `EXPECT_EQ` in those cases. Less verbose and should actually be better performance-wise (the general matcher code invoked here is heavier from inspection of GTest some while ago).

I prefer `EXPECT_THAT` only for complex matching (like contains a substring etc.).

A matter of preference, but this is my rationale.

Reply:
@MatthiasKillat Agree, was just following the convention in the file. I will change this one though.